src/components/deploy-table.tag

```diff
@@ -0,0 +1,31 @@
+<DeployTable>
+
+  Service: <input name="service" onkeyup={ edit } placeholder="Try Media"/>
+  <table class="table">
+    <thead>
+      <tr>
+        <th>Name</th>
+        <th>Build Time</th>
+        <th>Status</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr each="{ item in list }">
```

Comment:
I'm not sure if we want to bother with this yet, but in the future Riot 4 is going to make keys a requirement for each loops. They also speed up loops a bit. Might not be a bad idea to add that. 

`<tr each="{ item, i in list }" key={i}>`